### ISSUE TITLE
new torrentleech issue? 

### ISSUE THREAD

**Issue by smorison** [SickChill/SickChill#4628]
not sure if SR uses its own packaging... nodejs details:
pip search nodejs                                                                                                   
nodejs (0.1.1)  - Python bindings and utils for Node.js and io.js                                                   
INSTALLED: 0.1.1 (latest) 

happening on my synology install DS918+    O/S DSM 6.1.6-15266 Update 1

SickRage Info:
Branch: master
Commit: a7602639bd525f27475f1eeb2fc504111a14b51c
Version: v2018.04.08-1
Database Version: 44.0
Python Version: 2.7.14 (default, Dec 14 2017, 15:51:29) [GCC 6.4.0]
SSL Version: LibreSSL 2.6.3
OS: Linux-4.4.15+-x86_64-with

13:08:45 ERROR::SEARCHQUEUE-MANUAL-81967 :: [TorrentLeech] :: [a760263] Request failed: Missing Node.js runtime. Node is required and must be in the PATH (check with `node -v`). Your Node binary may be called `nodejs` rather than `node`, in which case you may need to run `apt-get install nodejs-legacy` on some Debian-based systems. (Please read t he cfscrape README's Dependencies section: https://github.com/Anorov/cloudflare-scrape#dependencies.               
Traceback (most recent call last):                                                                                  
  File "/app/sickrage/sickbeard/helpers.py", line 1525, in handle_requests_exception raise requests_exception                                                                                        
EnvironmentError: Missing Node.js runtime. Node is required and must be in the PATH (check with `node -v`). Your Node binary may be called `nodejs` rather than `node`, in which case you may need to run `apt-get install nodejs-legacy` on some Debian-based systems. (Please read the cfscrape README's Dependencies section: https://github.com/Anorov/cloudflare-scrape#dependencies.  
```

**Comment by ajankuv** [comment]
happening on custom hw, same errors

**Comment by ajankuv** [comment]
node is not a python module, make sure its installed system wide.

IE: apt-get install nodejs-legacy

**Comment by smorison** [comment]
hi Ajankuv,
nodejs is install system wide already... and the synology install of linux doesn't come with apt-get on it, so i'm bit lost as to how to proceed.

pip search nodejs                                                                                                   
nodejs (0.1.1)  - Python bindings and utils for Node.js and io.js                                                   
  INSTALLED: 0.1.1 (latest)

**Comment by ajankuv** [comment]
duh, i read that and had a brainfart

Start by logging into DSM and going to Package Manager
Install the "Node.js v8" package or whichever is the lateste at the time. (Located under "Developer Tools")

I found the above from someones docs on gist.

**Comment by paladinpizza** [comment]
I am on same version, platform Windows 10.  Above response does not resolve.  Receiving the same error message, specific to TL.  Installed nodejs runtime, still receiving same issue.  Install done as administrator.  SR runs as administrator.

**Comment by smorison** [comment]
Ajankuv,
added v8 and that didn't work for me, can you point me to the docs on git and i'll go annoy someone else :)

**Comment by f4780y** [comment]
Having the same issue on OSX. Seems to be a cloudflare challenge issue with Torrentleech?

**Comment by ajankuv** [comment]
For me it ended up being node not installed, I run Ubuntu so an apt-get fixed it.

**Comment by luchtgitaar** [comment]
I also had this issue. Apt-get nodejs-legacy (and just to be sure the other cloudflare scrape dependencies), but now Sickrage does not find any new or old episodes. 

Anyone else with the same problem?

**Comment by hannesbr** [comment]
Cant find torrents either, installed nodjs-legacy. No errors

**Comment by carbonneaud** [comment]
Perhaps your issue is related to https://github.com/SickRage/SickRage/issues/4649

**Comment by busemann** [comment]
I'm on Windows 10 and had the same problem after installing SickRage using the windows installer file.
I solved my problem by manually installing both Python and SickRage using [this](https://www.htpcguides.com/install-sickrage-windows-usenet-torrent-tv/) guide.
NOTE: I did not get SickRage to work with latest Python 3 version. Had to install latest Python 2 version.
I also have Node.js installed globally, NOT nodejs-legacy.
